study.md

```diff
@@ -28,16 +28,24 @@ Please name two things the DOM is not.
 
 ```md
 <!-- your answer here -->
+1)The DOM is not a set of data structures, it's a object model that specifies interface
+2)The DOM is not an HTML webpage, it is an API that lets one interact with the HTML/XML webpage
+
+source: https://www.w3.org/TR/DOM-Level-1/introduction.html
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
 <!-- your answer here -->
+To a certain extent; both behave in similar ways in the sense that they're both dynamic
+ the DOM is more of a tree diagram that represents the different content inside a HTML doc
 ```
```

Comment:
Good points here. What we see in Dev Tools provide a great visual representation of the DOM.